docs/guides/development-guides/how-to-show-and-hide-a-split-view-pane-with-mvvm.md

```diff
@@ -3,17 +3,304 @@ id: how-to-show-and-hide-a-split-view-pane-with-mvvm
+        Title="how_to_show_and_hide_a_split_view_pane_with_mvvm"
+
+		xmlns:convs="clr-namespace:how_to_show_and_hide_a_split_view_pane_with_mvvm.Models"
+		>
+```
+Now add the converter to the static resources of the Window by doing the following.
+``` xml
+<Window.Resources>
+    <convs:SplitViewIconConverter x:Key="SplitViewIconConverter"/>
+</Window.Resources>
+```
+Bind the button's content to the converter and pass the IsSplitViewPaneOpen property as a parameter to the converter.
+``` xml
+<Button Command="{Binding ChangeSplitViewPaneStatusCommand}">
+    <Button.Content>
+        <MultiBinding Converter="{StaticResource SplitViewIconConverter}">
+            <Binding Path="IsSplitViewPaneOpen"/>
+        </MultiBinding>
+    </Button.Content>
+</Button>
+```
+<br>
+The final MainWindow code should look like this:
+
+``` xml
+<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:vm="using:how_to_show_and_hide_a_split_view_pane_with_mvvm.ViewModels"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="how_to_show_and_hide_a_split_view_pane_with_mvvm.Views.MainWindow"
+        x:DataType="vm:MainWindowViewModel"
+        Icon="/Assets/avalonia-logo.ico"
+        Title="how_to_show_and_hide_a_split_view_pane_with_mvvm"
+
+		xmlns:convs="clr-namespace:how_to_show_and_hide_a_split_view_pane_with_mvvm.Models"
+		>
+
+    <Design.DataContext>
+        <!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+        <vm:MainWindowViewModel/>
+    </Design.DataContext>
+
+	<Window.Resources>
+		<convs:SplitViewIconConverter x:Key="SplitViewIconConverter"/>
+	</Window.Resources>
+	
+		<SplitView PanePlacement="Right" DisplayMode="CompactInline" IsPaneOpen="{Binding IsSplitViewPaneOpen}">
+			<SplitView.Pane>
+				<StackPanel VerticalAlignment="Top" Margin="5" Orientation="Horizontal">
+					<Button Command="{Binding ChangeSplitViewPaneStatusCommand}">
+						<Button.Content>
+							<MultiBinding Converter="{StaticResource SplitViewIconConverter}">
+								<Binding Path="IsSplitViewPaneOpen"/>
+							</MultiBinding>
+						</Button.Content>
+					</Button>
+					<TextBlock VerticalAlignment="Center" Margin="15, 0, 0, 0" FontSize="25" Text="Settings"/>
+				</StackPanel>
+			</SplitView.Pane>
+			<TextBlock HorizontalAlignment="Center" VerticalAlignment="Center" Text="MainWindow" FontSize="25"/>
+		</SplitView>
+</Window>
+```
+:::warning
+If you want to approach the problem by using just a normal converter and passing the IsSplitViewPaneOpen property to it as parameter, don't try to do that since ConverterParameter does not currently support binding. 
```

Comment:
The converter itself definitely supports bindings. I am not sure what the issue is here?

Reply:
Converter yes, but not the parameter. From what I found online, specifically [here](https://github.com/AvaloniaUI/Avalonia/issues/6840), while Converter can be binded, ConverterParameter can't